hs/Scripts.hs

```diff
@@ -71,7 +71,9 @@ graphScripts = do
          "/static/js/graph/sidebar/focus_descriptions.js",
          "/static/js/common/export/export.js"])
     H.script ! A.type_ "text/jsx" ! A.src "/static/js/common/react_modal.js.jsx" $ ""
-
+    H.script ! A.type_ "text/jsx" ! A.src "/static/js/graph/react_node.js.jsx" $ ""
```

Comment:
The `toScript` function appears to be gaining ambiguity here, and this section of code is not consistent. Should we remove the `toScript` function, or create a sister `test/jsx` `toScript` function as well?